web/beacon-app/src/application/routes/root.tsx

```diff
@@ -52,7 +52,6 @@ const router = createBrowserRouter(
         <Route path="dashboard" element={<Home />} />
         <Route path="projects">
           <Route index element={<ProjectsPage />} />
-          <Route path=":project-setup" element={<>Project setup</>} />
```

Comment:
The route has been removed since it's not used and it was causing a blank page with `Project setup` to appear instead of the `ProjectDetailPage` in dev.

Reply:
Nice catch again!